types/react-loadable/test/index.tsx

```diff
@@ -82,3 +82,28 @@ const used300 = <Loadable300 text='300' />;
 const used400 = <Loadable400 text='400' />;
 
 Loadable100.preload();
+
+// dynamic import syntax confuses tslint@5.5.0, so some disable comments were necessary.
+// all of these disables are specifically for working around this issue,
+// no rules were intended to be skipped.
+
+// tslint:disable-next-line one-variable-per-declaration
+const WithImport = Loadable({
+  loader: () => import('./imports/with-default'),
+  loading: LoadingComponent // tslint:disable-line semicolon
+  // render is optional
+}); // tslint:disable-line semicolon
```

Comment:
The linter is working for me locally even after removing these disables. Could you try refreshing your `node_modules`?